public/app/themes/justice/src/patch/js/ccfw-storage.js

```diff
@@ -1,29 +1,82 @@
 /**
  * A script for determining our storage method
  *
- * Falls back to cookies if localStorage is unavailable
+ * Falls back to cookies if localStorage is unavailable, otherwise
+ * we disable storage all together.
  */
-if (window.localStorage === undefined) {
-    Storage = {
-        getItem: (key) => {
-            const value = `; ${document.cookie}`;
-            const parts = value.split(`; ${key}=`);
+let Storage = {
+    disabled: true
+}
+
+const CCFWStorage = {
+    /**
+     * Both local() and cookie() return booleans
+     * false = invalid
+     * true = valid
+     * First we test for localStorage, if this is false, we test for cookies
+     * If cookie returns true, valid() returns true, otherwise, valid() returns
+     * false.
+     *
+     * @returns {boolean}
+     */
+    valid: () => {
+        return CCFWStorage.local() || CCFWStorage.cookie()
+    },
+    local: () => {
+        let mod = 'ccfw';
```

Comment:
I think this should be assigned with `const` vs `let`.

what do you think about moving this declaration to the top of the file, then also use it for the cookie test, to save a few bytes.

e.g. 

```js
document.cookie = `${mod}=1`
```

Reply:
Cool point :) although, in this case, I'd like to keep the checks independent. Using a const for both makes sense. Thank you.